rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -1017,6 +1017,49 @@ private void logUpdateCollections(Transaction rskTx) {
         eventLogger.logUpdateCollections(sender);
     }
 
+    protected void processValidationFailure(Federation proposedFederation) {
+        eventLogger.logCommitFederationFailure(rskExecutionBlock, proposedFederation);
+        logger.warn("[processValidationFailure] Proposed federation validation failed so svp values will be cleared.");
```

Comment:
Should we log something else? The block number where it failed maybe? We could then look for the event in that block

Reply:
good idea yes

Reply:
updated